CONTRIBUTING.md

```diff
@@ -253,6 +253,15 @@ steps.
 * Wait for a project maintainer to make an approving review.
 * Wait for your pull request to be merged.
 
+For CI you will need to ping the oneAPI Construction Kit reviewers by adding
+@ock-reviewers as a comment (or @ock-workflow-reviewers if it is workflow
+related). The CI can then be approved based on the PR. Note that currently some
```

Comment:
This shouldn't be necessary, I thought? https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#about-code-owners says "Code owners are automatically requested for review when someone opens a pull request that modifies code that they own." If that does not work yet, I think we are doing something wrong.

Reply:
ah fair, maybe just keep the ping for merging?

Reply:
fixed

Reply:
Looks right, though it's something that's effectively an extra bullet point in the list just above. Not sure if there's a nice unobtrusive way of putting it in there.

Reply:
it feels too big a paragraph to add in there - it does mention fixing test failures in the bullet point, so I think I'd rather leave it as is.